config/root-config.in

```diff
@@ -76,11 +76,10 @@ rootlibs="-lCore -lImt -lRIO -lNet -lHist -lGraf -lGraf3d -lGpad -lROOTVecOps -l
           -lRint -lPostscript -lMatrix -lPhysics -lMathCore -lThread\
           -lMultiProc"
 
-if [ "@UNIX@" != "1" ] || [ "@CMAKE_SIZEOF_VOID_P@" != "4" ]; then
+if [ "@dataframe@" == "ON" ]; then
```

Comment:
We can use the suggestion below for now, but we should have a generic solution if more things are allowed to be OFF in the future:
```suggestion
if echo "${features}" | grep -q "dataframe"; then
```

Reply:
Meh, Stefan didn't enable pushing by repo owners. I'll merge (after amending) by hand.